.github/workflows/release.yaml

```diff
@@ -40,3 +40,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - run: gh workflow run ci.yml --repo nextstrain/docker-base
+  create-github-release:
+    needs: [run]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.version}}
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.10'
+    - name: Download extract-release-notes.py from Nextclade repo
+      run: curl -OL https://github.com/nextstrain/nextclade/raw/master/scripts/extract-release-notes.py
```

Comment:
This seems like unnecessarily tight coupling between Augur and Nextclade.

Reply:
I agree, maybe it can be placed somewhere like `nextstrain/.github/scripts/extract-release-notes.py`. Or do you have a better idea here?

Reply:
I think its premature to standardize around this. I would likely copy that short program into Augur as `devel/extract-release-notes` and let it change over time uncoupled from Nextclade. For example, something will need to change to account for the `## __NEXT__` marker in Augur's changelog.

If I was feeling saucy, I'd replace the script with a short Perl one-liner:

```
perl -ne 'print if /^## (?!__NEXT__)/ ... (/^## / and exit)'
```

(complete with fix for `## __NEXT__`)

Reply:
@tsibley point taken from https://github.com/nextstrain/cli/commit/27d1ef7a19b344073646ae19f2728b6e5ae0c86b, parsing changelog is not a good solution here. Would be better to somehow have one source which feeds into both GH Release + changelog, but that requires extra tweaking of the release process.

To keep things simpler, I updated the GitHub Release to instead link to the changelog file:

![augur-example-github-release](https://user-images.githubusercontent.com/13424970/172495459-2b312f82-9f61-4270-af90-0ae75ff95efd.png)

Reply:
Updated with a new script `devel/changes` ported over from https://github.com/nextstrain/cli/pull/186.